qiskit/circuit/library/standard_gates/equivalence_library.py

```diff
@@ -439,6 +442,33 @@
 def_sdg.append(U1Gate(-pi / 2), [q[0]], [])
 _sel.add_equivalence(SdgGate(), def_sdg)
 
+# CSGate
```

Comment:
Are these equivalences actually necessary? Since these are the `definition`s of the gates, I thought we get these translations for free 🙂 @kdk would probably know best!

Reply:
Since now `CSGate` and `CSdgGate` are defined using the `CPhaseGate`, should we keep their equivalences?

Reply:
I think that it will be better to keep them now. @Cryoris - what do you think?

Reply:
Agreed 👍🏻